src/optimizer-page/scan-results/BrokenLinkTable.tsx

```diff
@@ -80,7 +80,7 @@ const BrokenLinkTable: FC<BrokenLinkTableProps> = ({
   return (
     <Card className="unit-card rounded-sm pt-2 pl-3 pr-4 mb-2.5">
       <p className="unit-header">{unit.displayName}</p>
-      <Table
+      <DataTable
```

Comment:
Since switching from `Table` to `DataTable` will lead to some visual differences, it'd probably be good to get some UX/Product input on the change.

I'll add another comment in this thread after I take a few comparison screenshots and play around with the `DataTable` a bit to see what options are available.